src/redux/actions/cellSets/updateCellSetSelected.js

```diff
@@ -12,7 +12,6 @@ const updateCellSetSelected = (experimentId, keys, tab) => async (dispatch, getS
   await dispatch({
     type: CELL_SETS_SET_SELECTED,
     payload: {
-      experimentId,
       keys,
```

Comment:
Can you remove the `experimentId` parameter from the function signature as well?